docs/cmdline_migration.md

```diff
@@ -46,6 +46,7 @@ You can use the following command-line options in OpenJ9, just as you did in Hot
 | [`-Xrs`](xrs.md)                                                 | Prevents the OpenJ9 run time environment from handling signals.                                                                              |
 | [`-Xss`](xss.md)                                                 | Sets the Java&trade; thread stack size. (Equivalent to `-XX:ThreadStackSize`). <i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Note:** Unlike HotSpot, this option applies only to the Java stack. OpenJ9 has a separate native stack for operating system threads (see [`-Xmso`](xmso.md))  |
 | [`-Xverify:mode`](xverify.md)                                    | Enables or disables the verifier.                                                                                                            |
+| [`-XX:ConcGCThreads`](xxconcgcthreads.md)                        | Configure number of GC mutator background threads.                                                                                           |
```

Comment:
Please say "Configures the number of ....." for parity with the rest of the table contents. Please also amend for the other 2 options added.

Reply:
See 
https://github.com/eclipse/openj9-docs/pull/464/commits/1bccc4d24e6b98d4966aa1e3492a812dda6eb391
https://github.com/eclipse/openj9-docs/pull/464/commits/0c09eaea9661f8789a01655e01df0c3d8173d6b3